js/main.js

```diff
@@ -107,107 +130,69 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 // switch to show or hide no entry text
-const $noEntries = document.querySelector('.center');
+const $noEntries = document.querySelector('#center');
 function toggleNoEntries() {
-  if (data.entries.length > 0) {
-    $noEntries.setAttribute('class', 'hidden center');
+  if (data.entries.length < 1) {
+    $noEntries.className = 'center';
   } else {
-    $noEntries.setAttribute('class', 'center');
+    $noEntries.className = 'hidden';
   }
 }
 
 // viewswapping between entries and entry-form pages
 function viewSwap(viewName) {
   if (viewName === 'entry-form') {
-    $entries.setAttribute('class', 'hidden');
-    $entryForm.setAttribute('class', 'display');
+    $entries.classList.add('hidden');
+    $entryForm.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
-  } else {
-    $entries.setAttribute('class', 'display');
-    $entryForm.setAttribute('class', 'hidden');
+  } else if (viewName === 'entries') {
+    $entryForm.classList.add('hidden');
+    $entries.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
   }
 }
 
 // shows Entries page when the Entries tab is clicked
 $nav.addEventListener('click', function () {
+  data.view = 'entries';
   viewSwap('entries');
 });
 
 // shows the Entry Form page when NEW button is clicked
 $newButton.addEventListener('click', function () {
+  $form.reset();
+  $imgSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $pageName.textContent = 'New Entries';
+  data.view = 'entry-form';
   viewSwap('entry-form');
 });
 
-// switch to Edit Entries page when pen icon is clicked
-$ul.addEventListener('click', function (event) {
```

Comment:
You should still have an eventListener on the `ul.` I think you removed this when you changed the event listener to only the pencil, but the eventListener should be on this element rather than the pencil